### DISCUSSION TITLE
system not upgrading

### DISCUSSION THREAD

**Discussion by kdmiller45** [mastodon/mastodon#18048] [General]
Trying Upgrading from 3.5.0 to 3.5.1
Ubuntu 18.04.6 LTS

I perform the following:
git fetch && git checkout v3.5.1
bundle install and yarn install
RAILS_ENV=production bundle exec rails assets:precompile
and restart all services

systemctl daemon-reload
systemctl start mastodon-web mastodon-sidekiq mastodon-streaming

and have even completely restarted the server and I am still at 3.5.0

**Comment by Gargron** [comment]
Systemctl start won’t restart already started services, use restart, and no daemon-reload necessary.
